src/tools/miri/src/lib.rs

```diff
@@ -16,6 +16,9 @@
 #![feature(unqualified_local_imports)]
 #![feature(derive_coerce_pointee)]
 #![feature(arbitrary_self_types)]
+#![cfg_attr(bootstrap,feature(legacy_receiver_trait))]
+#![feature(unsigned_is_multiple_of)]
+#![feature(extract_if)]
```

Comment:
Why are you adding new feature gates here?

Reply:
Merge error. Working on it but in meetings now...